code/game/machinery/computer/_computer.dm

```diff
@@ -45,17 +52,55 @@
 		icon_state = initial(icon_state)
 		update_icon()
 
+/obj/machinery/computer/proc/can_smooth()
+	// Determine whether we can smooth and form a console bank.
+	var/list/states = icon_states(icon)
+	var/teststate = icon_state + "-c"
+	return teststate in states
```

Comment:
If someone in ten years looks at this PR and goes "wow, he merged THAT shitcode?", they should know that I merged it *ironically*.